### DISCUSSION TITLE
Trying to run REI on a server with NBT Crafting caused a crash, solution?

### DISCUSSION THREAD

**Discussion by AsrielDewitt** [shedaniel/RoughlyEnoughItems#1007] [Q & A] [answered]
As the title says. I'm trying to run the server with REI and NBT Crafting, it causes a crash and it suggest me to use rei_server,rei_client or rei_common.

**Comment by shedaniel** [comment, marked as answer]
You should ask NBT Crafting to fix it, this is a requirement since 1.19.
